docker/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: '3.1'
+services:
+
+  nginx:
+    image: nginx:1.15.7-alpine
+    container_name: nginx
+    volumes:
+      - wordpress_data:/var/www/html/:ro
+      - ./etc/nginx/conf/nginx.conf:/etc/nginx/conf/nginx.conf:ro
+      - ./etc/nginx/conf.d:/etc/nginx/conf.d:ro
+    ports:
+      - 80:80
+      - 443:443
+    links:
+      - wordpress:php
+      - wp-db:db
+
+  wordpress:
+    image: wordpress:5.0.0-php5.6-fpm-alpine
+    container_name: wiki
+    restart: always
+    links:
+      - wp-db:mysql
+    expose:
+      - 9000
+    volumes:
+      - wordpress_data:/var/www/html/
+      - ./etc/wordpress/themes:/var/tmp/wp-themes:ro
+      - ./etc/wordpress/plugins:/var/tmp/wp-plugins:ro
+      - ./etc/wordpress/entrypoint.sh:/usr/local/bin/docker-entrypoint.sh
+    environment:
+      WORDPRESS_DB_HOST: mysql
+      WORDPRESS_DB_PASSWORD: 123456
+      WORDPRESS_CONFIG_EXTRA: |
+        /* defaults */
+        define('WP_DEFAULT_THEME', 'quads');
+
+  wp-db:
+    image: mariadb
+    container_name: wiki_db
+    restart: always
+    volumes:
+      - wiki_db:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=123456
+
+  quads:
+    depends_on:
+      - quads-db
+    build: .
+    container_name: quads
+    ports:
+      - 8080:8080
+    environment:
+      - MONGODB_IP=quads-db
+      - WORDPRESS_IP=wordpress
+
+  quads-db:
+    image: mongo:4.0.4-xenial
```

Comment:
What do you think about using an Alpine image instead for mongo?  Probably lighter-weight  https://github.com/mvertes/docker-alpine-mongo

Reply:
Good point, not sure if the bindings will be the same but I'll check